src/overlays/actors/ovl_En_Bjt/z_en_bjt.c

```diff
@@ -15,10 +15,37 @@ void EnBjt_Destroy(Actor* thisx, PlayState* play);
 void EnBjt_Update(Actor* thisx, PlayState* play);
 void EnBjt_Draw(Actor* thisx, PlayState* play);
 
-void func_80BFDA48(EnBjt* this, PlayState* play);
-void func_80BFDAE8(EnBjt* this, PlayState* play);
+void EnBjt_Talk(EnBjt* this, PlayState* play);
+void EnBjt_Wait(EnBjt* this, PlayState* play);
+
+#define TOILET_HAND_STATE_TALKING (1 << 3)   // Actually talking to Player
+#define TOILET_HAND_STATE_TEXTBOX (1 << 4)   // whenever a textbox is on screen
+#define TOILET_HAND_STATE_APPEARING (1 << 7) // Appearing
+#define TOILET_HAND_STATE_VANISHING (1 << 8) // Vanishing
+#define TOILET_HAND_STATE_VISIBLE (1 << 9)   // Out
```

Comment:
```suggestion
#define TOILET_HAND_STATE_TALKING (1 << 3)   // Actually talking to Player
#define TOILET_HAND_STATE_TEXTBOX (1 << 4)   // Whenever a textbox is on screen
#define TOILET_HAND_STATE_APPEARING (1 << 7) // Appearing
#define TOILET_HAND_STATE_VANISHING (1 << 8) // Vanishing
#define TOILET_HAND_STATE_VISIBLE (1 << 9)   // Out
```
Just for consistency